packages/api/server/http.mts

```diff
@@ -45,6 +46,11 @@ router.use(express.json());
 
 router.options('/file', cors());
 
+router.options('/examples', cors());
```

Comment:
This is a typo I think?
Can we call the endpoint `ai/apps/edit` ? There will be another (no streaming) that will be `ai/apps/create` that I'll add on top